ksql-rest-model/src/main/java/io/confluent/ksql/rest/entity/LagInfoEntity.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2020 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.rest.entity;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.errorprone.annotations.Immutable;
+import java.util.Objects;
+
+@Immutable
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class LagInfoEntity {
```

Comment:
note to self : this class is pretty much same as LagInfo in streams.. but we need all these jackson annotations.. so had to create a new one

Reply:
There's that and also the fact that the constructor of LagInfo is not public.  If you could make those changes to the original object, this could be removed.